index.rst

```diff
@@ -505,15 +505,117 @@ on the producer end.
 Conclusions
 -----------
 
-TO BE EXTENDED AND ORGANIZED IN TICKET DM-13307.
-
-For most experiments and metrics measured, assuming that the size of alerts has been estimated accurately,
-there is a significant impact on Kafka compute utilization and
-timing when postage stamp cutouts are included in the alert packets.
-The inclusion of postage stamp cutouts is currently a science requirement.  However,
-results here suggest that additional consideration be
-given to alternatives for the distribution of stamps, if it is possible
-that the current requirements may be adjusted.  One alternative to including postage
+Compute Resources and Settings Recommendations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+For a robust alert distribution service, a cluster of at least three Kafka brokers and a replication factor of two
+is recommended.  However, we did not test here the performance effects of a replication factor greater than one.
+
+Kafka is a memory intensive application.  In the experiments here, each Kafka broker tends to utilize all
+available memory, which it needs to buffer active producers and consumers.
+In terms of hardware recommendations, Confluent, the main contributor to Kafka, recommends a machine with
+64 GB of RAM or an AWS instances with 32 GB.  An appropriate back of the envelope calculation to determine
+hardware needs is `write_throughput`*30 if you want a 30 second buffer.  If we assume one alert is ~135 KB
+and possible bursts of up to 10,000 alerts/second, that is equivalent to 1.35 GB/second.  If we want to buffer
+for between 30-60 seconds, given the time between visits, an appropriate memory range is then between ~40-80 GB.
```

Comment:
Above, you say we “buffer active producers and consumers”, whereas this reads like we're simply maintaining a buffer of all the events received. Are these actually the same thing?

(I read “buffering consumers” as maintaining a separate per-consumer buffer, for example).